src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Utility/ProjectInstalledPackage.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+
+namespace NuGet.PackageManagement.VisualStudio.Utility
+{
+    internal class ProjectInstalledPackage
+    {
+        public VersionRange AllowedVersions { get; }
+        public PackageIdentity InstalledPackage { get; }
+        public ProjectInstalledPackage(VersionRange versionRange, PackageIdentity installedPackage)
```

Comment:
nitpick: Methods (including constructors) should always have a line of whitespace. Removing whitespace between auto-properties and fields are ok, but that's because they're 1 liners.